frontend/src/components/viewers/MetricsVisualizations.tsx

```diff
@@ -289,3 +329,42 @@ function buildConfusionMatrixConfig(
     },
   ];
 }
+
+interface ScalarMetricsSectionProps {
+  artifact: Artifact;
+}
+function ScalarMetricsSection({ artifact }: ScalarMetricsSectionProps) {
+  const customProperties = artifact.getCustomPropertiesMap();
+  const name = customProperties.get('name')?.getStringValue();
+  const data = customProperties
+    .getEntryList()
+    .map(x => ({
+      key: x[0],
+      value: JSON.stringify(getMetadataValue(customProperties.get(x[0]))),
+    }))
+    .filter(metric => metric.key !== 'name');
+
+  if (data.length === 0) {
+    return <></>;
+  }
+  return (
+    <>
+      {
+        <div className={padding(40, 'lrt')}>
+          <div className={padding(40, 'b')}>
+            <h3>{'Scalar Metrics: ' + name}</h3>
+          </div>
+          <PagedTable
+            configs={[
+              {
+                data: data.map(d => [d.key, d.value]),
+                labels: ['name', 'value'],
+                type: PlotType.TABLE,
+              },
+            ]}
+          />
+        </div>
+      }
+    </>
+  );
```

Comment:
nit the extra layer of indentation seems unnecessary

```suggestion
  return (
        <div className={padding(40, 'lrt')}>
          <div className={padding(40, 'b')}>
            <h3>{'Scalar Metrics: ' + name}</h3>
          </div>
          <PagedTable
            configs={[
              {
                data: data.map(d => [d.key, d.value]),
                labels: ['name', 'value'],
                type: PlotType.TABLE,
              },
            ]}
          />
        </div>
  );
```

Reply:
Thank you! Applying the change